rpc/src/lib.rs

```diff
@@ -93,3 +93,7 @@ pub mod mayastor {
 pub mod csi {
     include!(concat!(env!("OUT_DIR"), "/csi.v1.rs"));
 }
+
+pub mod registration {
+    include!(concat!(env!("OUT_DIR"), "/v1.registration.rs"));
```

Comment:
The CSI path has the version number after the module name. Should we keep them all consistent?

Reply:
We surely can, I kept it the way its for control-plane. We would need to change first in `mayastor-api` for that.

Reply:
That's fine. Leave it as it is for now.